tests/Makefile.js

```diff
@@ -0,0 +1,381 @@
+/**
+ * @fileoverview Release tests for the Makefile.js
+ * @author Victor Hom
+ */
+
+"use strict";
+
+const assert = require("chai").assert;
+const shell = require("shelljs");
```

Comment:
I'm not sure if it's possible at all, but if it is, it would be a lot better to mock file structure through `mockfs`. Just don't know if it would affect shelljs.